client.go

```diff
@@ -170,9 +167,9 @@ func (ce *Client) Serve(ctx context.Context) {
 			if isClosed(ce.done) {
 				return
 			}
-
-			// If we have enough sessions, we wait for error or done signal.
-			if ce.SessionCount() >= ce.conf.MinSessions {
+			// If MinSessions is set to 0 then we connect to all available servers.
```

Comment:
Do we set min sessions for Skywire dmsg client to a value higher than 0? Otherwise all clients will connect to all servers which is not needed. Only service clients should connect to all servers as they are central to the network functioning.

Reply:
Yes, the default min value is 1 that we use in most places (there is a variable called `DefaultMinSessions`). 0 was not an option before, it was auto changed to 1 if it was ever set. I have removed that part of the node now. Also, 0 is now only used for services.